package.json

```diff
@@ -23,11 +23,9 @@
   "author": "Astronomer, Inc.",
   "license": "ISC",
   "peerDependencies": {
-    "eslint-config-airbnb": "^18.2.0"
-  },
-  "devDependencies": {
     "babel-eslint": "^10.1.0",
-    "eslint": "^7.7.0",
-    "eslint-config-airbnb": "^18.2.0"
+    "eslint": "^7.15.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-jest": "^24.1.3"
```

Comment:
@samblackk @ryanahamilton I figured adding this as peerDependencies would be easier to maintain so where ever we use this package we can insure the same versions.  That way we don't need to install all of them directly in ever repo.  Does this go against best practices?

Reply:
I think maybe this should be dependencies.  I was misunderstanding the use-case for peerDependencies.  I'll make an update to this PR.

Reply:
These would be dependencies :)